src/game_inventory.cpp

```diff
@@ -1,21 +1,53 @@
 #include "game.h"
 
+#include "cata_utility.h"
 #include "inventory_ui.h"
 #include "player.h"
 #include "item.h"
 #include "itype.h"
 
+item_location inv_internal( player &p, const inventory_selector_preset &preset,
+                            const std::string &title, int radius,
+                            const std::string &none_message )
+{
+    p.inv.restack( &p );
+    p.inv.sort();
+
+    inventory_pick_selector inv_s( p, preset );
+
+    inv_s.set_title( title );
```

Comment:
> In which case you actually need a virtual std::string title() const method.

That would require inheriting it every time I need a custom hint or title: doesn't worth it.

> 100% of your current instances set title so you can and should do this via the constructor.

Looks like we can't tell each other anything new on that matter. I'd prefer to stick to the "properties" in this very case for the reasons stated above.